hack/release/common.sh

```diff
@@ -8,7 +8,9 @@ config(){
   RELEASE_REPO_ECR=${RELEASE_REPO_ECR:-public.ecr.aws/${ECR_GALLERY_NAME}/}
   RELEASE_REPO_GH=${RELEASE_REPO_GH:-ghcr.io/${GITHUB_ACCOUNT}/karpenter}
 
-  PRIVATE_PULL_THROUGH_HOST="${AWS_ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com"
+  PRIVATE_HOST="${AWS_ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com"
```

Comment:
```suggestion
  SNAPSHOT_HOST="${AWS_ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com"
```

nit: naming consistency to show these concepts are related. This helps when someone comes in later and is trying to parse concepts that go together

Reply:
I think `PRIVATE_HOST` is appropriate since it's used for both the pull through cache based replicas as well as the snapshot repos. We've discussed dropping the replicas offline so I think it would make the most sense to rename it in a follow up PR.

Reply:
I still disagree but not gonna block on it 👍🏻